remediation/fence-agents-remediation/fence-agents-remediation.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Fence Agents Remediation
+nav_order: 3
+has_children: true
+parent: Remediation
+---
+
+## Fence Agents Remediation
+
+Generally Available
+{: .label .label-green }
+
+[Fence Agents Remediation](https://github.com/medik8s/fence-agents-remediation#readme) (FAR) is a Kubernetes operator that uses external tools to *fence* unhealthy nodes. The tools are a set of well-known [upstream fence agents (FA)](https://github.com/ClusterLabs/fence-agents), where each one of them can be used for different environments, to fence a node using a reboot action. Moreover, FAR can be used by the [Node HealthCheck Operator](https://github.com/medik8s/node-healthcheck-operator#readme) (or any other operator that supports the external remediation API) to remediate an unhealthy node.
+
+FAR is not only fencing a node when it becomes unhealthy, it also *remediates* the node from being unhealthy to healty again. It adds a taint to evict stateless pods, fence the node with a fence agent and fully remediate it with resource deletion to remove any remaining workloads (mostly statefull workloads) after reboot.
+
+FAR is recommended when a node becomes unhealthy, and we want to completely isolate the node from a cluster, since we can’t “trust” the unhealthy node, to prevent it from accessing the shared resources like [RWO volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes). The operator is available in the Kubernetes community, [OperatorHub.io](https://operatorhub.io/operator/fence-agents-remediation), and can be installed manually as mentioned in [FAR documentation](https://github.com/medik8s/fence-agents-remediation#installation)
+
+As opposed to other remediation systems, FAR requires credentials/access to a management interface (e.g., Intelligent Platform Management Interface ([IPMI](https://en.wikipedia.org/wiki/Intelligent_Platform_Management_Interface)), etc.) or an API to reboot a node, but it has many advantages.
+
+### Advantages
+
+* Robustness - FAR can remediate an unhealthy node using a management interface (e.g., IPMI) while still keeping control plane connectivity.
```

Comment:
> while still keeping control plane connectivity

what do you mean with this? 🤔

Reply:
The term `control plane` might be confusing in our world, as it could hint to Kubernetes control plane node, but it wasn't my intention.
I meant that by using fence agent to reboot the node we receive an acknowledgment that the fence agent finished/failed/succeeded while still maintaining a connection to the node, and not losing it during/after the reboot process.

Reply:
>I guess what you are trying to point out here is: we have direct feedback if the fence action succeeded or not, right?

Correct

>We connect to the "management interface".

After your [last comment](https://github.com/medik8s/docs/pull/63#discussion_r1345695572) about the difference between `management interface` and  `traditional API`, I went with only using the API option. So unless you suggest using the other  `management interface` or explaining what this interface is earlier on this page, then I would prefer to use the API term.

>Robustness - By using a management interface, FAR has direct feedback about the result of the fence action without having to use the Kubernetes API.

Better, but I went with something else to keep a similar pattern/structure to the advantages bullets (begin the sentence with `FAR...`), and use `API` over the `management interface`.